pydoc-markdown/src/test/test_processors/test_google.py

```diff
@@ -70,3 +70,32 @@ def test_google_processor(processor=None):
       value = scanner.advance()
   ```
   ''')
+
+  assert_processor_result(processor or GoogleProcessor(),
+  '''
+  Args:
+    s (str): A string.
+    b (int): An int.
+  Examples:
+    ```
+    print("Hello World")
+    ```
+  Returns:
+    any: Something funny.
+  ''',
+  '''
+  **Arguments**:
+
+  - `s` _str_ - A string.
+  - `b` _int_ - An int.
+
+  **Examples**:
+
+  ```
+      print("Hello World")
```

Comment:
@ariG23498 Is the indentation here intentional? Seems off to me

Reply:
Yes here we cannot take the indentation spaces under consideration. I would love to know if you had something in mind to change this behaviour?

Reply:
Hey @ariG23498 ,

I would expect the result to be rendered like in the docstring:

`````
  Examples:
    ```
    print("Hello, World")
    ```
`````

Instead of


`````
  Examples:
  ```
      print("Hello, World")
  ```
`````

Reply:
Hey @NiklasRosenstein 
I have left working on this
And am not going to in the near future. You can close the PR if I am blocking the process 😄

Reply:
I feel like this is a fairly easy thing to fix, but before doing it I must make sure of one thing:

@NiklasRosenstein You say you expect
```
    **Examples:**

    ```
    print("Hello, World")
    ```
```
 but that would render a *quote* bloc instead of a code bloc in most editors, like so

    ```
    print("Hello, World")
    ```

Maybe you meant
````
    **Examples:**

```
print("Hello, World")
```
````

?